fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/PathNotFoundExceptionMapper.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.http.commons.exceptionhandlers;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.fcrepo.kernel.api.exception.PathNotFoundException;
+import org.slf4j.Logger;
+
+/**
+ * Catch PathNotFoundException
+ *
+ * @author robyj
+ */
+@Provider
+public class PathNotFoundExceptionMapper implements
+        ExceptionMapper<PathNotFoundException>, ExceptionDebugLogging {
+
+    Boolean showStackTrace = true;
```

Comment:
this variable doesn't appear to be used, i think you can remove it and its setter

Reply:
agreed. i'm removing the line plus the function that uses it